index.js

```diff
@@ -156,6 +156,9 @@ async function createCodeDeployDeployment(codedeploy, clusterName, service, task
   let codeDeployGroup = core.getInput('codedeploy-deployment-group', { required: false });
   codeDeployGroup = codeDeployGroup ? codeDeployGroup : `DgpECS-${clusterName}-${service}`;
 
+  let codeDeployDescription = core.getInput('codedeploy-deployment-description', { required: false });
+  codeDeployDescription = codeDeployDescription ? codeDeployDescription : '';
```

Comment:
As mentioned in the [action.yml](https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/200/commits/0aab2bbf304e9e79621294f1ed598c1ca95147c4#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R32) the default value of `codedeploy-deployment-description` should be in `'DgpECS-{cluster}-{service}'`  format and not `''`(empty) right? If I am understanding it correctly.

Reply:
It's been a while since I did it, but I think I just screwed up the description in the action.yml... possibly copying and pasting the deployment group name 😬 . Looking again at the logic, if the description isn't set by a user then it shouldn't be passed to the api call at all.